src/css/math.less

```diff
@@ -298,6 +298,21 @@
     }
   }
 
+  .mq-limit {
+    text-align: center;
+
+    span  {
+      display: block;
```

Comment:
yeah, it's better, the class was already on this element, so I just changed CSS, it looks like this now: 
<img width="215" alt="screen shot 2016-03-10 at 5 12 57 pm" src="https://cloud.githubusercontent.com/assets/1845230/13661061/95e86a88-e6e3-11e5-8a23-97abad65298b.png">